### PR TITLE
Bugfix/1167/posting a local link hides the information

### LINKED ISSUE #1167: Posting a "local" link hides the information
### Steps
1. In a conversation post a link from a local server, e.g. http://nextcloud21.local/apps/files/img/add-color.png which is my dev instance but not available with that fake domain outside of the maschine.
2. In the web UI I simply see a link to `http://nextcloud21.local/apps/files/img/add-color.png`
3. The android app only shows a blank space because it can't generate a preview for the link:
Screenshot_2021-04-27-09-33-44-941_com nextcloud talk2

In this case it should fall back to a text message showing the link

### PR DESCRIPTION
**Pull request by timkrueger**
Resolves #1167

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -269,10 +269,6 @@ dependencies {
     implementation 'com.github.nextcloud:PopupBubble:1.0.6'
     implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
 
-    implementation('eu.medsea.mimeutil:mime-util:2.1.3', {
-        exclude group: 'org.slf4j'
-    })
```
Comment (AndyScherzinger) [inline app/build.gradle:274]: ❤️

**Review by AndyScherzinger — APPROVED** [review]

**Comment by timkrueger** [comment]
@AndyScherzinger The DCO check fails. I think because the error count for findbugs was decreased automatically. Can I set the DCO check to passed manually in that case?

**Comment by AndyScherzinger** [comment]
> Can I set the DCO check to passed manually in that case?

Absolutely, yes please :) 👍

**Comment by timkrueger** [comment]
Now the Merging is blocked by that unsigned commit. What can I do about that?

**Comment by AndyScherzinger** [comment]
> Now the Merging is blocked by that unsigned commit. What can I do about that?

@mahibi and I can still hit that button ;)

I raised the DCO issue with @tobiasKaminsky @ https://github.com/nextcloud/android/issues/8694 while that might not fix the signing issue. @tobiasKaminsky can we get the drone commits properly signed? (not sure if it is just the sign-off or if we need it pgp-signed)

**Comment by tobiasKaminsky** [comment]
> I raised the DCO issue with @tobiasKaminsky @ [nextcloud/android#8694](https://github.com/nextcloud/android/issues/8694) while that might not fix the signing issue. @tobiasKaminsky can we get the drone commits properly signed? (not sure if it is just the sign-off or if we need it pgp-signed)

They are properly signed, you can see as there is a green ✔, but email address is not valid, therefore DCO complains.
I'll try to solve it in nextcloud/android#8694 and once I have a solution, I'll post it to Talk :)

**Pull request merged**